### PR TITLE
Generates a random NIR number (fr_FR)

### PR DESCRIPTION
**Pull request by Ultim4T0m**
File : `Faker\Provider\fr_FR\Person`
New function : `nir($gender = null, $formatted = false)`

Generates a NIR number.

### CODE REVIEW

**Comment by fzaninotto** [comment]
Neat! Can you rebase? Also, could you add "social security" or "securité sociale" in the phpDoc to make it more searchable? "NIR" isn't very well-known.

File: src/Faker/Provider/fr_FR/Person.php [hunk 1]
```diff
@@ -67,4 +67,63 @@ public static function prefix()
...
+        $nir .= $department;
+
+        // Town number, depends on department length
+        if (strlen($department) === 2) {
+            $nir .= $this->numerify('###');
+        } elseif (strlen($department) === 3) {
+            $nir .= $this->numerify('##');
+        }
+
+        // Born number (dependending of town and month of birth)
```
Comment (vlakoff) [inline src/Faker/Provider/fr_FR/Person.php]: typopo "dependending"

**Comment by Ultim4T0m** [comment]
I rebased and added some tests.
Can someone rerun the build that failed ? 

I opened an Issue for the test in failure #1119

**Comment by Ultim4T0m** [comment]
Well, I rebased again, this time it's ok

**Comment by fzaninotto** [comment]
Thanks!

**Comment by Ultim4T0m** [comment]
Perfect, thank you!

**Pull request merged**